### ISSUE TITLE
Code Snipet not shown when test fails

### ISSUE THREAD

**Issue by alexviz7** [Qytera-Gmbh/cypress-xray-plugin#87]
In continuation to the issue 79, a made a test fail on purpose to see how the results will be shown.
No code code snipet is shown just a scrennshot is uploaded with no further information.
![2023-06-26 18_30_56-Execute Test TC-2451 - DevStack Jira — Mozilla Firefox](https://github.com/Qytera-Gmbh/cypress-xray-plugin/assets/9295838/29d9b3a6-0691-446a-b553-fefcf21d2c75)

**Comment by csvtuda** [comment]
Thanks for creating a new issue. I will look into it to see what I could do.

**Comment by csvtuda** [comment]
I just released version 4.0.0. Cucumber results should now display code snippets with screenshots attached to individual steps:
![image](https://github.com/Qytera-Gmbh/cypress-xray-plugin/assets/18561435/4b03d309-9e8e-495f-8053-077153f50560)

Please let me know if this fixed your issue.

**Comment by alexviz7** [comment]
it works now for single scenario
![Screenshot 2023-07-31 162836](https://github.com/Qytera-Gmbh/cypress-xray-plugin/assets/9295838/7b684e3d-8d60-479b-bec0-303bab0849af)

**Comment by alexviz7** [comment]
Yes I m bit confused too, I see 3 screenshots and 3 fails on the scenario outline when i  had set the fail fast to 2 fails not threee. Console log also shows 2 fails and the cucumber reporter as well.

**Comment by alexviz7** [comment]
Its like it created the last screenshot twice.

**Comment by csvtuda** [comment]
Can you share the actual Cucumber JSON report or does it contain sensitive data? It would help a lot.

**Comment by alexviz7** [comment]
Yes unfortunately I can't give you all of the report. However even the screenshots folder has 4 screenshots (run mode has a retry, so each fail has to fail twice).
![Screenshot 2023-07-31 173422](https://github.com/Qytera-Gmbh/cypress-xray-plugin/assets/9295838/f43867ca-cb28-42af-9300-5a603a5ba404)

**Comment by alexviz7** [comment]
Ok removing the 1 retry had as a result that only 1 screenshot is made. I think its not an issue and ticket can be closed.
